### PR TITLE
YLW 3111: 3. Implement conditional statements and loops

### PR DESCRIPTION
**Pull request by davidgreenland**
### What
Implementing conditional statements and loops

### Why
To understand loops and conditionals in C#.

### How
Implemented for loop, while loop and do-while loop printing the numbers 1 to 10 to the console. Checked if a number was even or odd using an if-else statement.

### CODE REVIEW

File: LearningCSharp/Program.cs [hunk 1]
```diff
@@ -9,4 +9,46 @@
...
+    Console.Write($"{k} ");
+    k++;
+} while (k <= 10);
+Console.WriteLine();
+Console.WriteLine();
+
+Console.WriteLine("A random number between 1 and 10 is generated.");
+Console.WriteLine("An if-else statement checks if a number is even or odd, and outputs the result:");
+Random rnd = new Random();
+int number = rnd.Next(1, 11);
```
Comment (SiddiqueAbdullah) [inline LearningCSharp/Program.cs]: change it to var

File: LearningCSharp/Program.cs [hunk 2]
```diff
@@ -9,4 +9,46 @@
...
+while (j <= 10)
+{
+    Console.Write($"{j} ");
+    j++;
+}
+Console.WriteLine();
+Console.WriteLine();
+
+Console.WriteLine("The following numbers are printed with a do-while loop:");
+
```
Comment (SiddiqueAbdullah) [inline LearningCSharp/Program.cs]: empty line not needed

File: LearningCSharp/Program.cs [hunk 3]
```diff
@@ -9,4 +9,46 @@
...
+var k = 1;
```
Comment (SiddiqueAbdullah) [inline LearningCSharp/Program.cs]: can reuse _j_

**Review by SiddiqueAbdullah — CHANGES_REQUESTED** [review]

File: LearningCSharp/Program.cs (hunk 1 above)
Comment (davidgreenland) [reply LearningCSharp/Program.cs]: Done.

File: LearningCSharp/Program.cs (hunk 2 above)
Comment (davidgreenland) [reply LearningCSharp/Program.cs]: Removed.

File: LearningCSharp/Program.cs (hunk 3 above)
Comment (davidgreenland) [reply LearningCSharp/Program.cs]: Done.

**Review by SiddiqueAbdullah — APPROVED** [review]

**Pull request merged**